documentation/slate/source/includes/objects.md

```diff
@@ -562,6 +562,18 @@ $ curl 'https://api.navitia.io/v1/coverage/sandbox/poi_types' -H 'Authorization:
 |id|string|Identifier of the poi type|
 |name|string|Name of the poi type|
 
+### Stands
+
+A description of the number of stands/places and vehicles available at one POI
+(for example a bike sharing station, or a car park).
+
+|Field           |Type|Description                                                              |
+|----------------|----|-------------------------------------------------------------------------|
+|available_places|int |Number of places where one can park (bike, car, etc.)                    |
+|available_bikes |int |Number of bikes available                                                |
+|total_stands    |int |Total number stands (occupied or not, with or without special equipment) |
+|status          |enum|Information about the station itself:<ul><li>`unavailable`: Navitia is not able to obtain information about the station</li><li>`open`: The station is open</li><li>`closed`: The station is closed</li></ul>|
+
```

Comment:
I don't find any attribut of Stands for parking_places. What about deleting 'car' from (bike, car, etc.)??

Reply:
Indeed, it looks like we didn't mutualize this... :disappointed: 
Will remove that.